samples/snippets/csharp/VS_Snippets_CLR_System/system.String.LastIndexOf/cs/lastindexof22.cs

```diff
@@ -1,59 +1,46 @@
-﻿// <Snippet22>
-using System;
+﻿using System;
 
 public class Example
 {
-   public static void Main()
-   {
-      int position = 0;
-      string s1 = "ani\u00ADmal";
-      string s2 = "animal";
-      
-      // Find the index of the soft hyphen.
-      position = s1.LastIndexOf("m");
-      Console.WriteLine("'m' at position {0}", position);
-      if (position >= 0)
-         Console.WriteLine(s1.LastIndexOf("\u00AD", position));
+    public static void Main()
+    {
+        // <Snippet22>
+        int position = 0;
+        string s1 = "ani\u00ADmal";
+        string s2 = "animal";
 
-      position = s2.LastIndexOf("m");
-      Console.WriteLine("'m' at position {0}", position);
-      if (position >= 0)   
-         Console.WriteLine(s2.LastIndexOf("\u00AD", position));
-      
-      // Find the index of the soft hyphen followed by "n".
-      position = s1.LastIndexOf("m");
-      Console.WriteLine("'m' at position {0}", position);
-      if (position >= 0)
-         Console.WriteLine(s1.LastIndexOf("\u00ADn", position));
+        // Find the index of the soft hyphen followed by "n".
+        position = s1.LastIndexOf("m");
+        Console.WriteLine("'m' at position {0}", position);

```

Comment:
Consider interpolated strings here:

```suggestion
        Console.WriteLine($"'m' at position {position}");

```